manual/adobeshockwaveplayer/adobeshockwaveplayer.nuspec

```diff
@@ -1,29 +1,31 @@
-<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>adobeshockwaveplayer</id>
-    <version>12.3.4.204</version>
+    <version>12.3.5.205</version>
+    <packageSourceUrl>https://github.com/AndreAugustoAAQ/chocolatey-packages/tree/main/adobeshockwaveplayer</packageSourceUrl>
+    <owners>André Augusto,pauby,dtgm</owners>

```

Comment:
Can you remove your name from the owners?

Reply:
done